test/unit/api/bitwise/bit_and/regular/bit_and.hpp

```diff
@@ -0,0 +1,94 @@
+//==================================================================================================
```

Comment:
Why did you have to write these tests? Didn't we have them for not 512?

Reply:
The functions were tested, not the operators.